draft-ietf-privacypass-auth-scheme.md

```diff
@@ -592,12 +592,9 @@ ensure that the token type is sufficiently clearly defined to be used for both
 token issuance and redemption, and meets the common security and privacy
 requirements for issuance protocols defined in {{Section 3.2 of ARCHITECTURE}}.
 
-This registry also will also allow provisional registrations to allow for
-experimentation with protocols being developed. Designated experts review,
-approve, and revoke provisional registrations.
-
-Values 0xFF00-0xFFFF are reserved for private use, to enable proprietary uses
-and limited experimentation.
+Values 0xFF00-0xFFFF are reserved for private use. Implementers can use values
+in this range for experimentation with new token type protocols, as well as other
+proprietary uses that do not require interoperability.
```

Comment:
Wonder if we should have an extra line that implementers should not rely on this being compatible on the public internet (since other private use implementations may collide). Though might be implicit in the "interoperability" bit.

Reply:
Right, that's the intent of that sentence.